src/JWT/Serializers/JsonSerializerFactory.cs

```diff
@@ -1,15 +1,22 @@
 ﻿namespace JWT.Serializers
 {
-    internal static class JsonSerializerFactory
+    public static class JsonSerializerFactory
     {
+        private static IJsonSerializer _jsonSerializer;
+
+#if MODERN_DOTNET
+        public static void SetSerializer(IJsonSerializer jsonSerializer)
+        {

```

Comment:
I think it should be a ctor, used by the builder and anyone else.

Reply:
I made the JsonSerializerFactory non-static and it looks better now.